nautobot/extras/tests/integration/test_plugin_banner.py

```diff
@@ -0,0 +1,35 @@
+from unittest import skipIf
+
+from django.conf import settings
+
+from nautobot.utilities.testing.integration import SeleniumTestCase
```

Comment:
Leave it for now, but we should start using `SplinterTestCase` and use the method calls for that instead. We want to move towards Splinter and refactor the few existing integration test cases to use that, so I think we should try to avoid creating new tests from the other one. Once we refactor everything, we can just replace `SeleniumTestCase` w/ `SplinterTestCase`.